deploy/111_deploy_universal_spokepool.ts

```diff
@@ -53,7 +61,22 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   // target address of the spoke pool. This is because the HubPool does not pass in the chainId when calling
   // relayMessage() on the Adapter. Therefore, if Universal SpokePools share the same address,
   // then a message designed to be sent to one chain could be sent to another's SpokePool.
-  await deployNewProxy("Universal_SpokePool", constructorArgs, initArgs);
+  const { proxyAddress } = await deployNewProxy("Universal_SpokePool", constructorArgs, initArgs);
+
+  const provider = hre.ethers.provider;
+  const safeCode = await provider.getCode(EXPECTED_SAFE_ADDRESS);
+  if (safeCode !== "0x" && proxyAddress) {
+    const factory = await hre.ethers.getContractFactory("Universal_SpokePool");
+    const contract = factory.attach(proxyAddress);
+
+    const owner = await contract.owner();
+    if (owner !== EXPECTED_SAFE_ADDRESS) {
+      await contract.transferOwnership(EXPECTED_SAFE_ADDRESS);
+      console.log("Transferred ownership to Expected Safe address:", EXPECTED_SAFE_ADDRESS);
```

Comment:
wdyt about waiting for transaction confirmation and then reading the updated value here?